alf/algorithms/on_policy_algorithm.py

```diff
@@ -13,52 +13,49 @@
 # limitations under the License.
 """Base class for on-policy RL algorithms."""
 
-from abc import abstractmethod
-
-import tensorflow as tf
+import torch
 
+import alf
 from alf.algorithms.rl_algorithm import RLAlgorithm
-from alf.data_structures import ActionTimeStep, Experience, TrainingInfo
-from alf.algorithms.off_policy_algorithm import OffPolicyAlgorithm
+from alf.data_structures import TimeStep, Experience, TrainingInfo
 
 
-class OnPolicyAlgorithm(OffPolicyAlgorithm):
+class OnPolicyAlgorithm(RLAlgorithm):
     """
     OnPolicyAlgorithm works with alf.drivers.on_policy_driver.OnPolicyDriver
     to do training at the time of policy rollout.
 
     User needs to implement rollout() and train_complete().
```

Comment:
```rollout()``` -> ```rollout_step()```

Currently there is a mis-match of the name for this function: in implementation it is ```rollout()```, in documentation it is a mixture of ```rollout()``` and ```rollout_step()```.